pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -77,6 +78,14 @@ stdenv.mkDerivation rec {
     sha256 = "pXrNDkSCmBcA/fhllsdBPLYe9H915HR/2oCegjG42Ww=";
   };
 
+  patchFlags = ["--binary"];
```

Comment:
What is this for?

Reply:
Absolutely nothing, me being dumb and doing this late at night the last two days :D